.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: python
 python:
  - 2.7
+ - 3.6
```

Comment:
Yeah, I added 3.6 to get around the weird error in 2.7, but that exploded the build matrix, so I tried to add the tox-travis plugin to make the integration smarter, but never got it properly configured. If 2.7 is working under 3.6, I think the right play is to pull out the tox-travis plugin and just tell travis to use 3.6